codegen-exports/src/lib.rs

```diff
@@ -1,5 +1,2 @@
-#[cfg_attr(feature = "stable", proc_macro_hack::proc_macro_hack)]
 pub use include_flate_codegen::deflate_file;
-
-#[cfg_attr(feature = "stable", proc_macro_hack::proc_macro_hack)]
 pub use include_flate_codegen::deflate_utf8_file;
```

Comment:
Could we just as well delete this reexport crate directly?

Reply:
Yes, I wasn't sure what that was for (if it was required before) so I left them. But I did run a test to see what would happen if I removed them, and everything worked.

It's fine to delete them and remove the `stable` feature entirely.

Reply:
#23